tests/swagger/test_swagger.py

```diff
@@ -1200,7 +1208,90 @@ def test_must_add_vpc_deny(self):
                         {"Fn::Sub": ["execute-api:/${__Stage__}/GET/foo", {"__Stage__": "prod"}]},
                     ],
                     "Effect": "Deny",
-                    "Condition": {"StringEquals": {"aws:SourceVpc": ["vpc-123"]}},
+                    "Condition": {
+                        "StringEquals": {"aws:SourceVpc": ["vpc-123"]}
+                    },
+                    "Principal": "*",
+                },
+            ],
+        }
+
+        self.assertEqual(deep_sort_lists(expected), deep_sort_lists(self.editor.swagger[_X_POLICY]))
+
+
+    def test_must_add_vpc_allow_string_and_instrinic(self):
+
+        resourcePolicy = {
```

Comment:
A couple changes:
- Please make sure the explicit intrinsic entries do NOT have "VPC" or "VPCE" in their strings, to ensure we don't regress on requiring regex matching.
- Please include more than one to ensure it's handling the full list.

Can we also test both whitelist and blocklist in a single test, to ensure both are applied?

Reply:
Updated intrinsic entries and added more test cases.